### DISCUSSION TITLE
Works like a charm, except for geolocation.

### DISCUSSION THREAD

**Discussion by IonutInit** [umami-software/umami#2524] [Q&A] [answered]
I've tried everything I could think of, but I'm not even able to spot the problem. Any advice on how to make the map work?

**Comment by mattegan** [comment, marked as answer]
Is your instance behind a reverse proxy of some sort? I have Umami proxied through Nginx and had to add `proxy_set_header X-Client-IP $remote_addr` in my `location` block. The headers that are used can be found in the [request-ip](https://www.npmjs.com/package/request-ip) readme.

**Comment by IonutInit** [comment]
Nice one, cheers. Worked instantly. Hopefully this will help others.
